### DISCUSSION TITLE
Can no longer list users in Administration Console after creating LDAP user storage provider

### DISCUSSION THREAD

**Discussion by christian-2** [keycloak/keycloak#20340] [Q&A] [answered]
I have added an LDAP user storage provider for the [online LDAP test server](https://www.forumsys.com/2022/05/10/online-ldap-test-server/) `ldap.forumsys.com` to a realm in a basic Keycloak 21.1.1 installation. 

Everything works fine, and I can now log into my OIDC client as user `nobel`, `curie`, etc. But if I try listing users in the Administration Console with _realm_ / `Users` (to see which were synced from LDAP) an error occurs: "undefined is not an object (evaluating 'x.config?.enabled[0]')". I can still list users (the user `admin`) in realm `master` without this error.

What could be wrong here? Presumably a field `enabled` is missing from some object, but [`ComponentRepresentation`](https://www.keycloak.org/docs-api/21.1.1/rest-api/index.html#_componentrepresentation) (which would appear to be the most obvious candidate in this context) has no such field.

**Comment by christian-2** [comment, marked as answer]
I have now been able to fix this by adding `-s 'config.enabled=["true"]'` when creating the LDAP user storage provider with `kcadm.sh create components -r $KEYCLOAK_REALM -s providerType=org.keycloak.storage.UserStorageProvider -s providerId=ldap ...`.

I've found this solution mentioned [here](https://lists.jboss.org/pipermail/keycloak-user/2018-February/012996.html). But where in the [documentation](https://www.keycloak.org/documentation) are these fields (`config.debug`, `config.enabled`, `config.priority`) mentioned?

**Comment by justin-stephenson** [comment]
Same issue here:  https://github.com/keycloak/keycloak/issues/22087

Adding `-s config.enabled==["true"]` works but seems to be a new requirement for keycloak 21 and 22+
